### PR TITLE
New note in spa

### PR DESCRIPTION
**Pull request by ArpitAgrawal001**
Here's a simple flow chart:
```mermaid
sequenceDiagram
    participant Browser
    participant Server
    participant SPA

    Browser->>Server: HTTP request for https://studies.cs.helsinki.fi/exampleapp/spa
    activate Server
    
    Server-->>Browser: SPA files (HTML, CSS, JS) are served
    deactivate Server
    
    Browser->>SPA: Browser receives SPA files
    activate SPA
    
    SPA-->>Browser: SPA initializes and renders
    deactivate SPA
    
    Browser->>Server: AJAX requests for initial data/resources
    activate Server
    
    Server->>SPA: Initial data/resources are served
    deactivate Server
    activate SPA
    
    SPA-->>Browser: User creates a new note
    deactivate SPA
    Note right of SPA: Note creation action triggers an AJAX request
    
    Browser->>Server: AJAX request to create a new note
    activate Server
    
    Server-->>Browser: Confirmation of successful note creation
    deactivate Server
    
    Browser->>SPA: SPA updates interface to display new note
    activate SPA
    
    SPA-->>Browser: Executed the excercise:New note in Single page app diagram
    deactivate SPA
    Note right of Browser: Exercise 0.6
```

### CODE REVIEW

**Comment by ArpitAgrawal001** [comment]
Sir should i do some modifications in my sequence diagram of Excercise 0.6 or is it ready to merge?

And sir should i delete rest of the feature branches?

File: Part0/Excercise_0.6.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    activate SPA
+    
+    SPA-->>Browser: User creates a new note
+    deactivate SPA
+    Note right of SPA: Note creation action triggers an AJAX request
+    
+    Browser->>Server: AJAX request to create a new note
+    activate Server
+    
+    Server-->>Browser: Confirmation of successful note creation
```
Comment (mavenik) [inline Part0/Excercise_0.6.md:32]: This does not talk about the kind of HTTP request (GET/POST) and the status code, response data, etc. Can you update the diagram to include this information?

**Review by mavenik — CHANGES_REQUESTED** [review]
@ArpitAgrawal001 I see that your previous exercises are deleted on this branch. Probably because the latest changes in `develop` weren't pulled? I also see that Exercise 0.4 has been deleted for some reason. We need it back.

File: Part0/Excercise_0.6.md (hunk 1 above)
Comment (ArpitAgrawal001) [reply Part0/Excercise_0.6.md:32]: ok sir will update this.

**Comment by ArpitAgrawal001** [comment]
> @ArpitAgrawal001 I see that your previous exercises are deleted on this branch. Probably because the latest changes in `develop` weren't pulled? I also see that Exercise 0.4 has been deleted for some reason. We need it back.

Sir but after i will merge this branch in develop , all my three excercises will be available there right?

**Comment by ArpitAgrawal001** [comment]
Because in my develop branch excercise 0.4 and excercise 0.5 is available.

And after i merge this branch i have to delete all the 3 feature branches right?

**Comment by mavenik** [comment]
We need to delete branches, not files. If you delete files, they will be deleted in any branch that you merge your active branch to.

**Comment by ArpitAgrawal001** [comment]
Sir so should i have to include excercise 0.4 and excercise 0.5 in my
feature branch?

**Comment by ArpitAgrawal001** [comment]
Along with excercise 0.6

**Pull request closed without merging**